.github/workflows/main.yml

```diff
@@ -59,12 +62,20 @@ jobs:
           opam exec -- sed -i ' ' tests/sources/*.elpi
           & "$Env:CYGWIN_ROOT/setup-x86_64.exe" -q -P time
 
+      - name: Satisfy depext of camlp5 on OSX workaround
+        if: runner.os == 'macOS'
+        run: |
+          cpan install IPC::System::Simple
+          cpan install String::ShellQuote
+
```

Comment:
With @chetmurthy 's changes, we should be able to drop this block:
```suggestion
```

@gares I remember you have other plans, but I hope you'd be willing to try it.